presto-spi/src/main/java/io/prestosql/spi/security/AccessDeniedException.java

```diff
@@ -321,6 +326,11 @@ public static void denyCreateViewWithSelect(String sourceName, ConnectorIdentity
         throw new AccessDeniedException(format("View owner '%s' cannot create view that selects from %s%s", identity.getUser(), sourceName, formatExtraInfo(extraInfo)));
     }
 
+    public static void denyAccessToViewBecauseOwnerLacksPrivileges(AccessDeniedException e)
```

Comment:
I don't think this public method makes sense here, as it's only usage is inside `ViewAccessControl`. The other methods are intended for use by any access control implementation. Instead, we can do this inline in `ViewAccessControl`.

Reply:
done.